TicTacToe/TicTacToe/UI.cs

```diff
@@ -22,10 +22,18 @@ public class UI {
         }
 
         public void PlaceMarker() {
-            string input = this.io.GetInput();
-            int move = Int32.Parse(input);
-            this.board.UpdateBoard(move);
-            BoardView();
+            try {
+                string input = this.io.GetInput();
+                int move = Int32.Parse(input);
+                this.board.UpdateBoard(move);
+                BoardView();
+            } catch(FormatException) {
+                this.io.Print(InvalidEntryPrompt());
+                PlaceMarker();
+            } catch(IndexOutOfRangeException) {
```

Comment:
While wondering whether Java could handle multiple exceptions in one `catch`, I found the answer to be yes for Java 7 and later: https://docs.oracle.com/javase/tutorial/essential/exceptions/catch.html. Note sure if this has changed in Java 9, though.

Reply:
Nice! I was able to find a c# equivalent.

Reply:
That's good, since I forgot this was C#! :-)